airbyte-migration/io/build.gradle

```diff
@@ -0,0 +1,8 @@
+plugins {
+    id 'java-library'
```

Comment:
since this is just a skeleton of the project, I created a separate issue https://github.com/airbytehq/airbyte/issues/1402 to properly connect it to the API endpoint